library/core/src/lib.rs

```diff
@@ -184,6 +184,7 @@
 #![feature(intra_doc_pointers)]
 #![feature(intrinsics)]
 #![feature(lang_items)]
+#![feature(let_else)]
 #![feature(link_llvm_intrinsics)]
 #![feature(macro_metavar_expr)]
 #![feature(min_specialization)]
```

Comment:
this has been stabilized and is no longer needed

Reply:
Fixed.